.travis.yml

```diff
@@ -1,8 +1,11 @@
 sudo: required
-language: java
+language: python
 jdk:
   - openjdk8
```

Comment:
Yes, I have removed installation of jdk8.

Reply:
This is so because for arch:arm64 we required language:python instead of java other wise this " - source ~/virtualenv/python3.6/bin/activate" starts failing in travis for arm64.

Reply:
jdk:
  - openjdk8

To clarify, I am asking regarding the above one. I got why we are taking python as a language.